LICENSE

```diff
@@ -269,6 +269,7 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
      (BSD 3 Clause) d3.min.js (https://github.com/mbostock/d3/blob/master/LICENSE)
      (BSD 3 Clause) DPark (https://github.com/douban/dpark/blob/master/LICENSE)
      (BSD 3 Clause) CloudPickle (https://github.com/cloudpipe/cloudpickle/blob/master/LICENSE)
+     (BSD 2 Clause) Zstd-jni (https://github.com/luben/zstd-jni/blob/master/LICENSE)
```

Comment:
ZStandard itself is also included, not just zstd-jni. We'd need an entry for it, and its LICENSE as below.
https://github.com/luben/zstd-jni/tree/master/src/main/native